app/controllers/users.rb

```diff
@@ -0,0 +1,15 @@
+class Chitter < Sinatra::Base
+  get '/users/new' do
+    erb :'users/new'
+  end
+
+  post '/users/new' do
+    user = User.new(name: params[:name], username: params[:username], email: params[:email], password: params[:password], password_confirmation: params[:password_confirmation])
```

Comment:
Metrics/LineLength: Line is too long. [176/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)